dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNLineInterfaceIIDM.cpp

```diff
@@ -308,11 +308,10 @@ LineInterfaceIIDM::getCurrentLimitPermanent(const std::string& season, CurrentLi
     default:
       return boost::none;
   }
-  unsigned int index = static_cast<unsigned int>(side) - 1;
   const auto& map = currentLimitsPerSeasonExtension_->getCurrentLimits();
   auto found = map.find(season);
-  if (found != map.end() && found->second.currentLimits.at(index)) {
-    return found->second.currentLimits.at(index)->permanentLimit;
+  if (found != map.end() && found->second.currentLimits.count(side) > 0 && found->second.currentLimits.at(side)) {
```

Comment:
How come that we don't have to do -1 anymore?

Reply:
It was a array before, now it is a map with the enum as key